3drepo/3drepo.io
# ISSUE #2958 - Custom scrollbars in V5

This fixes #2958 

#### Description
- Added custom scrollbars for V5.
- Scrollable area control created as ScrollArea control.
- Scrollbars added to all modals, to all dashboard pages, and to revision details (when clicking on container in container dashboard)


#### Test cases
- Open all modals and observe that scroll works as intended.
- Open all dashboard pages and observe that scroll works as intended.
- View a container's revisions using a mixed number of revisions (The list should reach a max of five revisions before scrolling).

#### Notes
- For most cases the contents of the scroll area did not need to be modified much. The exception to this is the revision details.
- The revisionsDetails pages are a little bit chaotic. In the styling a lot of values depend on other values (i.e. they're sums of padding and margin). Could it be useful define variables at the top of the page like I've done with the list item height?
- I wasn't sure how large to make the max height of the modals. I ended up using 70% of the screen height, but this can change.
